### PR TITLE
Add OpenMensa Metadata

### PR DESCRIPTION
**Pull request by jpbernius**
Related to #24

### CODE REVIEW

File: fmi-bistro/meta.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <tuesday open="07:00-19:00" />
+      <wednesday open="07:00-19:00" />
+      <thursday open="07:00-19:00" />
+      <friday open="07:00-19:00" />
+      <saturday closed="true" />
+      <sunday closed="true" />
+    </times>
+    <feed name="full" priority="1">
+      <schedule dayOfMonth="*" dayOfWeek="1" hour="8" retry="60 5 1440" />
+      <url>http://srehwald.github.io/eat-api/fmi-bistro/feed.xml</url>
```
Comment (srehwald) [inline fmi-bistro/meta.xml]: will feed.xml be created by #24 ?
what about https?

Same for ipp

Comment (jpbernius) [reply fmi-bistro/meta.xml]: Fixed https URLs.
Yes, `feed.xml` is generated by #24

**Comment by srehwald** [comment]
Alright, will merge it, once #24 is ready to merge

**Comment by jpbernius** [comment]
Should be good to go?

**Comment by srehwald** [comment]
was just waiting for travis build

**Comment by jpbernius** [comment]
FYI: https://openmensa.org/c/773

**Pull request merged**